### DISCUSSION TITLE
Wo ist die Einstellung:  B statt H?

### DISCUSSION THREAD

**Discussion by olsching** [thetwom/Tuner#42] [Q&A]
Hallo, wo und wie kann ich vom deutschen "H" auf das internationale "B" umstellen?

MfG olsching

**Comment by thetwom** [comment]
Diese Möglichkeit wurde erst kürzlich mit Version [v5.4.0](https://github.com/thetwom/Tuner/releases/tag/v5.4.0) eingeführt. Falls du fdroid verwendest, dort dauert es meist ein paar Tage, bis das verfügbar ist.

Grundsätzlich dann unter Einstellungen > Schreibweise.

**Reply by olsching** [reply]
Vielen Dank für die rasche Antwort!
Ja, ich habe die Äpp über F Droid installiert und noch die Version 5.3.

**Reply by vocna** [reply]
Why I can't see pitch H even on v4.x.x? And can't see the difference in notation between v4.x.x and v5.4.0?

**Reply by thetwom** [reply]
H is used only in certain languages ...
